src/content/docs/jellyfin-integration/configuring-shokofin.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: Shokofin Configuration
+sidebar:
+  order: 2
+Description: Shokofin Configuration
+---
+
+{/*
+    TODO: Write an overview for this section.
+
+    TODO: EVERYTHING. Everything in here is a literal copy/paste of the descriptions directly in Shokofin's settings page.
+
+    TODO: Add Revam's changes for VFS library management 
+*/}
+
+Before you start using the plugin it is important to connect the plugin to a running instance of **Shoko Server**. So head over to the plugin settings and provide the `Host`, `Username` and `Password` credentials and click the `Connect` button.
+
+![Shokofin - Connection Settings](/src/assets/img/shokofin/Shokofin-Connection-Settings.png)
+
+After you've established a connection to a running instance of **Shoko Server** then you're free to either use the plugin as-is or to change any of the other settings to your liking.
+
+## Connection Settings
+
+Host:
+
+    - This is the URL leading to where Shoko is running. You can input a full url, or just the dns name/ip.
+
+Username:
+
+    - The user should be an administrator in Shoko, preferably without any filtering applied.
+
+Reset the connection.
+
+    - Be sure to stop any tasks using this plugin before you press the button.
+
+## Metadata Settings
+
+Main title source:
+
+    - How to select the main title for each item. The plugin will fallback to the default title if no title can be found in the target language.
+
+Alternate title source:
+
+    - How to select the alternate title for each item. The plugin will fallback to the default title if no title can be found in the target language.
+
+Description source:
+
+    - How to select the description to use for each item.
+
+Allow any title in selected language.
+
+    - Will add any title in the selected language if no official title is found.
+
+Add all metadata for multi-episode entries.
+
+    - Will add the title and description for every episode in a multi-episode entry.
+
+Cleanup AniDB descriptions
+
+    - Remove links and collapse multiple empty lines into one empty line
+
+Minimalistic AniDB descriptions
+
+    - Remove any lines starting with '* ', '-- ', '~ ', 'Note', 'Source', and/or 'Summary'
+
+## Plugin Compatibility Settings
+
+We can optionally set some ids for interoperability with other plugins.
+
+Add AniDB IDs
+
+    - Enabling this will add the AniDB ID for all supported item types where an ID is available.
+
+Add TvDB IDs
+
+    - Enabling this will add the TvDB ID for all supported item types where an ID is available.
+
+Add TMDB IDs
+
+    - Enabling this will add the TMDB ID for all supported item types where an ID is available.
+
+## Library Settings
+
+If you wish to set the Series/Grouping setting below to "Don't group Series into Seasons", you must disable "Automatically merge series that are spread across multiple folders" in the settings for all Libraries that depend on Shokofin for their metadata.
```

Comment:
the merge option isn't required when the VFS is used, but it might still be recommended if the VFS is not used but shoko groups are used to create the show entries.